NOTICE

```diff
@@ -0,0 +1,22 @@
+Envoy
+Copyright (c) 2016-2017 Lyft Inc.  All rights reserved.
+
+Licensed under Apache License 2.0.  See LICENSE for terms.
+
+Zipkin (Apache V2)
```

Comment:
Yeah you're probably right, given those items are not included in the source itself. Might be worth have a note about other software that's included, for explicitness - it could just go in the README or something.